SubEthaEdit-Mac/Source/TCMHost.m

```diff
@@ -19,7 +19,7 @@
 void myCallback(CFHostRef myHost, CFHostInfoType typeInfo, const CFStreamError *error, void *myInfoPointer);
 
 
-@interface TCMHost (TCMHostPrivateAdditions)
+@interface TCMHost ()
```

Comment:
Yes, generally speaking that is a good change. The old "private additions" category names can almost all be gone. Also (but not shure if we should do it in the same step), moving the ivars from the @interface to the Interface Extension is also preferred. There is no place anymore for ivars in Headers unless absolutely needed by weird old code constructs.

Reply:
I think moving IVars (and possible removing other "PrivateAdditions" categories) should  definitely happen in a separate PR :)